src/usb.rs

```diff
@@ -68,4 +86,5 @@ unsafe impl UsbPeripheral for Peripheral {
 ///
 /// As this MCU family has only USB peripheral,
 /// this is the only possible concrete type construction.
-pub type UsbBusType = UsbBus<Peripheral>;
+pub type UsbBusType<DP = PA11<AF14<PushPull>>, DM = PA12<AF14<PushPull>>> =
+    UsbBus<Peripheral<DM, DP>>;
```

Comment:
Using this type for `stm32f303xd` devices with the set defaults is not possible / leads to a compiler error, am I understanding that correctly? What about providing different defaults for these devices or a different type alias?

Reply:
I've made a separate definition for the d/e devices.  It won't lead directly to a type error since `PA12<AF14<PushPull>>>` is still a valid type on the d/e devices, but it would be impossible to construct one :-)